module/VuFind/src/VuFind/RecordDriver/SolrMarc.php

```diff
@@ -72,4 +75,69 @@ public function getURLs()
             $this->getIlsURLs()
         );
     }
+
+    /**
+     * Takes a Marc field that notes are stored in (ex: 950) and a list of
+     * sub fields (ex: ['a','b']) optionally as well as what indicator
+     * numbers and values to filter for and concatenates the subfields
+     * together and returns the fields back as an array
+     * (ex: ['subA subB subC', 'field2SubA field2SubB'])
+     *
+     * @param string $field    Marc field to search within
+     * @param array  $subfield Sub-fields to return or empty for all
```

Comment:
You should reflect nullability in the comment to match the signature:

```suggestion
     * @param ?array $subfield Sub-fields to return or empty for all
```